lib/screens/config/v2/triptych.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Screens.Config.V2.Triptych do
+  @moduledoc false
+
+  alias Screens.Config.V2.{EvergreenContentItem, OLCrowding}
+
+  @type t :: %__MODULE__{
+          ol_crowding: OLCrowding.t(),
+          evergreen_content: list(EvergreenContentItem.t())
+        }
+
+  defstruct ol_crowding: nil,
+            evergreen_content: []
+
+  use Screens.Config.Struct, children: [ol_crowding: OLCrowding]
```

Comment:
I think `evergreen_content: {:list, EvergreenContentItem}` needs to be added to `children` here. That may be what Dialyzer is complaining about.

Reply:
```suggestion
  use Screens.Config.Struct, children: [ol_crowding: OLCrowding, evergreen_content: {:list, EvergreenContentItem}]
```